app/assets/stylesheets/styles/_footer.scss

```diff
@@ -0,0 +1,27 @@
+footer.page-footer {
+  bottom: 0;
+  width: 100%;
+  background-color: #333;
+  box-shadow: 0 0 3px #444;
+  color: #fff;
+  //margin-top: 65px;
+  margin-bottom: 0 !important;
+
+
+  .footer-links {
+    ul {
+      li {
+        display: inline-block;
+        padding-right: 15px;
+        padding-left: 15px;
+      }
+    }
+    a {
+      color: #fff;
+    }
+  }
+
+  .copyright {
+    background-color: #444;
```

Comment:
Color literals like `#444` should only be used in variable declarations; they should be referred to via variable everywhere else.